src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/CSharpFormattingOptions2.cs

```diff
@@ -229,6 +229,42 @@ private static Option2<T> CreateOption<T>(OptionGroup group, string name, T defa
         CSharpSyntaxFormattingOptions.Default.WrappingKeepStatementsOnSingleLine)
         .WithPublicOption(PublicFeatureName, "WrappingKeepStatementsOnSingleLine");
 
+    public static Option2<bool> WrapCallChains { get; } = CreateOption(
+        CSharpFormattingOptionGroups.Wrapping, "csharp_wrap_call_chains",
+        defaultValue: false);
+
+    public static Option2<bool> IndentWrappedCallChains { get; } = CreateOption(
+        CSharpFormattingOptionGroups.Wrapping, "csharp_indent_wrapped_call_chains",
```

Comment:
these shoudl likely be dotnet_ options.

NOte: we already have dotnet wrapping options like:

```
    internal static readonly Option2<OperatorPlacementWhenWrappingPreference> OperatorPlacementWhenWrapping = CreateOption(
        CodeStyleOptionGroups.ExpressionLevelPreferences,
        "dotnet_style_operator_placement_when_wrapping",
        defaultValue: OperatorPlacementWhenWrappingPreference.BeginningOfLine,
        serializer: new(
            parseValue: str => OperatorPlacementUtilities.Parse(str, OperatorPlacementWhenWrappingPreference.BeginningOfLine),
            serializeValue: OperatorPlacementUtilities.GetEditorConfigString));
```

So this should likely all go together and be consistent with that.